projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -143,37 +143,36 @@ def create_job(name: str, team: str, path: str, local: bool, cloud: bool):
         return f"Job with name {name} was created."
 
     @staticmethod
-    def create_deployment(name: str, team: str, path: str, reason: str, enabled: bool):
+    def create_deployment(name: str, team: str, path: str, reason: str):
         """
         Execute `Deploy job`.
         :param name: the name of the data job that will be deployed
         :param team: the team of the data job that will be deployed
         :param path: the path to the job's directory
         :param reason: the reason of deployment
-        :param enabled: flag whether the job is enabled (that will basically un-pause the job)
         :return: output string of the operation
         """
+        import io
+        import sys
+
+        buffer = io.StringIO()
+        sys.stdout = buffer
```

Comment:
Note that generally Redirecting sys.stdout is considered less than ideal practice and is saved for last resort. 

* sys.stdout is a global variable, and redirecting it affects the entire application. This can lead to unexpected results if different parts of the codebase are assuming different things about where stdout is going.
* In multi threads , which I think we might have a case here, redirecting stdout can cause problems, as threads might interfere with each other's output
 * If an error occurs after redirecting stdout but before it's reset back to its original value, well that would be an issue. 
  
  It's risky and you should be always careful when doing it. 
  
  In this case a better approach is to pass it another printer 
  
  You can create new printer easily : just a class decorated with `@printer` (as [seen here](https://github.com/vmware/versatile-data-kit/blob/ffdee8bac07e3718299eb10e024c7319179b3b93/projects/vdk-control-cli/src/vdk/internal/control/utils/output_printer.py#L63) ) 
  
  And then you can set the output = "memory" for example (something that stores the results in memory instead of printing to stdout ) for example.

Reply:
I have created a memory printer please check it